components/css/newsletter-text-only/_contentPage.scss

```diff
@@ -0,0 +1,173 @@
+$white:#ffffff;
+$black:#000000;
+$light-gray:#aaaaaa;
+$dark-gray:#2e2e2e;
+$gold:#c28e0e;
+
+.intro {
+	background-color: #c28e0e;
+	padding: 1.5em;
+
+
+	&__title {	
+	font-size: 2em;
+	line-height: 1.3em;
+	color: $black;
+	text-align: center;
+	text-transform:uppercase;
+	padding: .5em 0;
+	}
+	
+	&__issue {
+	font-size: 1em;
+	line-height: 1em;
+	color: $black;
+	text-align: center;
+	padding: 0;
+	}
+	
+	&__issue::after {
+	content:" ";
+	display: block;
+	width: 30%;
+	height: 3px;
+	margin: .3em auto !important;
+	background-color: $white;	
+	}
+	
+	&__description {
+	font-size: .9em;
+	line-height: 1.4em;
+	color: $black;
+	text-align: center;
+	padding: 1.5em 0 0 0;
+	}
+}
+
+
+.section {
+	background-color: $dark-gray;
+	padding: 1.5em;
+	
+	&__heading {
+	font-family:Impact,"Helvetica Neue",Helvetica,sans-serif;
```

Comment:
I forgot to tell you how to implement Champion. We'll do that next.

Reply:
```scss
&__heading{
    font-family:$font-champion-heavy;
}
```
This variable is declared within the `_fonts.scss` mixin.